_targets.R

```diff
@@ -13,6 +13,8 @@ library(tidyr)
 library(glue)
 library(epinowcast)
 library(baselinenowcast)
+library(forecasttools)
```

Comment:
What is this used for? Its not in the packages you pass to targets? 

Does the description need updating as well?

Reply:
oh let me remove

Reply:
nah its out of description I think